src/routes/Dashboard/components/Dashboard.scss

```diff
@@ -5,9 +5,18 @@
 .tabs {
   width: 100%;
   height: 600px;
+  overflow: scroll;
 }
 
 .map {
   width: 100%;
   height: 600px;
 }
+
+pre {
```

Comment:
Why not just go ahead and remove the tabs and raw data thing altogether. I just threw the data up temporarily until a map got added. You can see loaded data in redux dev tools.